### PR TITLE
Making sure travis does not fail

### PR DESCRIPTION
**Pull request by evert**

### CODE REVIEW

**Comment by DominikTo** [comment]
With https://github.com/fruux/sabre-katana/commit/071f32c00eaff9301a1a6ec13337a15f69aafc40 tests are passing now.

Two exceptions though:
* `--prefer-lowest` tests fail. Looks like some older dependency is breaking the `katana://` schema (see [here](https://travis-ci.org/fruux/sabre-katana/jobs/59870768)).
* `hhvm` tests fail (see [here](https://travis-ci.org/fruux/sabre-katana/jobs/59870775)).

/cc @evert

File: .travis.yml [hunk 1]
```diff
@@ -3,10 +3,34 @@ php:
...
 
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: 7
+
+env:
+  matrix:
+    - LOWEST_DEPS="" TEST_DEPS=""
+    - LOWEST_DEPS="--prefer-lowest" TEST_DEPS="tests/Sabre/"
```
Comment (Hywan) [inline .travis.yml]: That's not correct, we don't have the `tests/Sabre/` directory. Moreover we don't use the `TEST_DEPS` variable so you can remove it.

File: composer.json [hunk 2]
```diff
@@ -40,7 +39,8 @@
     },
     "suggest": {
         "ext-pdo_sqlite": "If you decide to use SQLite.",
-        "ext-pdo_mysql" : "If you decide to use MySQL."
+        "ext-pdo_mysql" : "If you decide to use MySQL.",
+        "ext-phar" : "If you plan to package sabre/katana as a phar archive"
```
Comment (Hywan) [inline composer.json]: Please, align `:` ;-).

File: lib/Stub/Phar.php [hunk 3]
```diff
@@ -20,6 +20,9 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
+ // @TODO: Workaround for
```
Comment (Hywan) [inline lib/Stub/Phar.php]: This is a workaround for a bug inside phpcs. What about disabling it temporarily?

**Comment by Hywan** [comment]
atoum is not compatible with HHVM yet. See https://github.com/atoum/atoum/pull/451.

I am working on the `--prefer-lowest` bug.

File: lib/Stub/Phar.php (hunk 3 above)
Comment (DominikTo) [inline lib/Stub/Phar.php]: This way we at least get phpcs for everything else, just not this file.

File: composer.json (hunk 2 above)
Comment (DominikTo) [inline composer.json]: Done. ;-P
https://github.com/fruux/sabre-katana/commit/8156c0621750f9c6ab903ba1202391725c2912b8

File: .travis.yml (hunk 1 above)
Comment (DominikTo) [inline .travis.yml]: Removed:
https://github.com/fruux/sabre-katana/commit/bdbc1e045171c07e527bb6edcd787f1dcbb1c686

File: composer.json [hunk 4]
```diff
@@ -20,9 +20,9 @@
         "sabre/dav"                : "2.2alpha4",
         "sabre/uri"                : "~1.0",
         "sabre/http"               : "4.0.0alpha1",
-        "hoa/core"                 : "~2.0",
+        "hoa/core"                 : "~2.0, >=2.15.04.13",
```
Comment (evert) [inline composer.json]: This should be ^2.15.04.13 for the same effect.

File: composer.json [hunk 5]
```diff
@@ -20,9 +20,9 @@
...
         "hoa/console"              : "~2.0",
-        "hoa/dispatcher"           : "~0.0",
+        "hoa/dispatcher"           : "~0.0, >=0.15.01.23",
```
Comment (evert) [inline composer.json]: This can be `^0.15.01.23` I believe

**Comment by evert** [comment]
Actually I'm not sure if `^` will work if hoa does not use semantic versioning. Will merge as-is =)

**Pull request merged**